testfiles/Fault_HumidistatOffset_Supermarket.idf

```diff
@@ -596,7 +596,7 @@
     ON/OFF,                  !- Schedule Type Limits Name
     Through: 12/31,          !- Field 1
     For:AllDays,             !- Field 2
-    Interpolate:Yes,         !- Field 3
+    Interpolate:Average,         !- Field 3
```

Comment:
This is interesting, I didn't know that `Interpolate:` was inside the `Through:` section so that you could presumably interpolate some sections and not others.  Weird.

Reply:
Presumably. I did not test but each interpolate is associated with a day schedule.